xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+    Create &ostack; system objects in the Cisco APIC.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Create &cloud; tenants and networks in the Cisco APIC using the Horizon dashboard.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Create a VXLAN-based network encapsulation in the Cisco ACI fabric.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Start &slsa; instances on different compute nodes that can ping their gateway and each other.
+    </para>
+
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ 
+ <sect1 xml:id="sec.prereqisites">
+  <title>Prequisites</title>
+    <para>
+     As the Cisco ACI fabric is customized and highly-variable per customer's needs, you must have your ACI fabric already set up and working, and you need your own ACI expert administrator.
+    </para>
+    <para>
+    If you are using Cisco UCS B series hardware there are additional prerequisites; please contact Cisco for information.
+    </para>
+    <para>
+    Start with a newly-installed &cloud; with all nodes discovered and allocated, and no barclamps yet deployed. (If you are using an existing installation then pay extra attention to all configurations, and check for duplicates and conflicts.)
+    </para>
+    <para>
+    Make sure the control node has an IP address that allows it to communicate to your Cisco APIC.
+   </para>
+    <para>
+     The examples in this document reference a simplified example network with these nodes:
+ </para>
+ 
+  <itemizedlist>
+   <listitem>
+    <para>
+        Two Cisco ACI leafs.
+    </para>
+   </listitem> 
+   
+   <listitem>
+    <para>
+        One admin node.
+    </para>
+   </listitem> 
+   
+    <listitem>
+    <para>
+        One control node.
+    </para>
+   </listitem> 
+   
+   <listitem>
+    <para>
+        Two compute nodes.
+    </para>
+   </listitem>  
+</itemizedlist>
+
+    <para>
+   The example network uses 192.168.124.0/24 for the admin network. Figure J-1 shows the topology of the example network.
+    </para>
+    
+      <figure>
+     <title>The example network topology.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="depl_opflex_1.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="depl_opflex_1.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+</sect1>
+
+    <sect1 xml:id="sec.networking">
+  <title>Configure Networking on Compute Nodes</title>     
+    <para>
+In the example network we have two compute nodes that are connected to our Cisco ACI fabric. Network configuration will be automated in future releases, but for now you must follow these steps to configure networking on all of your compute nodes.
+    </para>
+    
+<procedure>
+    <step>
+        <para>
+            Enter <literal>YAST | System | Network Settings</literal> (Click Continue when you see the Chef client warning).
```

Comment:
Replace: `<literal>YAST | System | Network Settings</literal>` -> `<menuchoice><guimenu>YaST</guimenu><guimenu>System</guimenu><guimenu>Network Settings</guimenu></menuchoice>` (Yes, it is more markup.) Also, Continue should be in a `<guimenu>` as well.